feat: gazelle_test test rule (#1785)
bazelbuild/bazel-gazelle: README.rst

After running ``update-repos``, you might want to run ``bazel run //:gazelle`` again, as the
``update-repos`` command can affect the output of a normal run of Gazelle.

> To verify that all generated is update-to-date, you can you ``gazelle_test`` rule.

Reviewer: ```suggestion
To verify that all BUILD files are update-to-date, you can use the ``gazelle_test`` rule.
```

.. code:: bzl

>   gazelle_test(

Reviewer: Please add a `load` statement.
